### ISSUE TITLE
[Issue]: Virtual memory card no longer works with Medal of honor Frontline

### ISSUE THREAD

**Issue by FascistMan** [emukidid/swiss-gc#873]
### Describe the bug

I had already played this title until reaching the last mission, just today I decided to continue my game and I was surprised that the game saved on the memory card is no longer recognized by Swiss.

### Steps to reproduce

Usually I enter the game without modifying the start at all (I use the PAL - ESP Version). The game was compatible with virtual memory cars in previous editions, but in this latest revision it is no longer available. It doesn't even allow me to move the game from the virtual memory card to the physical memory card so I can continue the game from there.

### Expected behavior

I just hope I can save my progress like before. And taking advantage of this, I would like to ask a question: How could I move a locked game from a virtual memory card to a physical one? The game has the same crash as F-Zero GX and Phantasy Star Online. (I don't know if it is possible to perform such a feat with GCMM.

In short, my medal of honor frontline save game is stuck on the virtual memory card, and I don't even have the possibility to move it to my physical memory

### Relevant log output

```shell
.
```

### Add screenshots

.

**Comment by Extrems** [comment]
You can use Dolphin's memory card manager to manage memory card images and import/export save files.

**Comment by FascistMan** [comment]
Solucionado, Gracias
